algorithm/index.js

```diff
@@ -1,17 +1,18 @@
-const loadRawDataFromDb = require('../db/loadRawDataFromDb');
-const writeFormattedDataToDb = require('../db/writeFormattedDataToDb');
-const { avgCalc, diffCalc } = require('./helpers');
+const {
+  loadRawData,
+  saveFormattedData,
+  avgCalc,
+  diffCalc,
+} = require('./helpers');
 
 module.exports = async () => {
-  const rawData = await loadRawDataFromDb();
+  const rawData = await loadRawData();
   const avgM2Price = avgCalc(rawData);
   const formattedData = rawData.map((obj) => {
     const formattedObj = {};
     formattedObj.thumbnail = obj.thumbnail;
```

Comment:
please change the syntax to a destructuring syntax e.i. const formattedObj = {thumbnail, price, size, country};